test/scripts/measures/2018/fixtures/expected-measures.json

```diff
@@ -14,6 +14,8 @@
     "lastPerformanceYear": null,
     "isHighPriority": true,
     "isInverse": true,
+    "isIcdImpacted": false,
+    "isToppedOutByProgram": false,
```

Comment:
Can we set one or more of these to be `true` (here in and in the test data) so that we can be sure it's actually parsing correctly and not just using a `false` default or something?

Reply:
@samskeller I updated the test data. The relevant test file seemed to be `expected-quality-measures.json`